csi/csi.go

```diff
@@ -0,0 +1,158 @@
+/*
+Copyright 2022 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package csi
```

Comment:
csi is a bit too generic, how about csi utils or csi manager?

Reply:
This will also change the folder name to csi-manager? if it's fine I can change it.

Reply:
As a file that contains code to create and maintain the resources required for a functional CSI driver, it might be appropriate to rename it to `csidriver`.

Also, shouldn't things like this go under something like `pkg/csi/`? Or is that out of fashion these days?